### PR TITLE
:memo: First try of contribution guidelines

### PR DESCRIPTION
**Pull request by cchampou**
This is the very first draft for our contribution guidelines.

Feel free to comment or ask any question about the content!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+the newly merged features. If you plan to work on a new feature or a bug fix,
+you need to create a new branch **starting from develop**, and call it after
+the subject of your work, then commit on this branch. When your work is ready
+to be reviewed, feel free to open a pull request.
+
+**Not a single commit should be pushed directly to `main` or `develop`.**
+
+## Pull requests
+
+> We highly recommend you to join the Discord server as mentioned in the first
```
Comment (M123-dev) [inline CONTRIBUTING.md:39]:
```suggestion
> We highly recommend you to join the [Discord server](https://discord.gg/AgNcUeS3Zx) as mentioned in the first
```

**Review by M123-dev — APPROVED** [review]
First review 🔥

**Review by Juliotati — APPROVED** [review]
LGTM

**Pull request merged**